src/main/java/com/porpapps/wealthwatchapi/model/Portfolio.java

```diff
@@ -1,16 +1,48 @@
 package com.porpapps.wealthwatchapi.model;
 
+import jakarta.persistence.*;
 import lombok.Data;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
 
 import java.time.LocalDateTime;
 
 @Data
```

Comment:
Also, add following annotation for each one of the models 
```java
@AllArgsConstructor
@NoArgsConstructor
```

Reply:
done.